.github/workflows/coding.yml

```
@@ -55,4 +56,74 @@
-          fail_ci_if_error: true
+          fail_ci_if_error: true
+
+  test-typescript-challenges:
+    runs-on: ubuntu-latest
+    needs: inspect-code
+    environment: CI
+    steps:
+      - name: Check out repository
+        uses: actions/checkout@v4
+        # with:
+          # path: programming/typescript/challenges
```

YAMLlint (1.35.1)

[warning] comment not indented like content

(comments-indentation)